spec/20-http_request_header_format.md

```diff
@@ -282,7 +282,7 @@ A `key` MUST begin with a lowercase letter or a digit and contain up to 256 char
 
 ##### Value
 
-The value is an opaque string containing up to 256 printable ASCII [[!RFC0020]] characters (i.e., the range 0x20 to 0x7E) except comma (`,`) and (`=`). Note that this also excludes tabs, newlines, carriage returns, etc.
+The value is an opaque string containing up to 256 printable ASCII [RFC0020] characters (i.e., the range 0x20 to 0x7E) except comma (,) and (=) which must end with a character which is not a space (0x20). Note that this also excludes tabs, newlines, carriage returns, etc. This means that all leading spaces MUST be preserved as part of the value and all trailing spaces are considered to be optional whitespace characters not part of the value. Optional whitespace MAY be excluded when propagating the header.
```

Comment:
```suggestion
The value is an opaque string containing up to 256 printable ASCII [[!RFC0020]] characters (i.e., the range 0x20 to 0x7E) except comma (,) and (=) which must end with a character which is not a space (0x20). Note that this also excludes tabs, newlines, carriage returns, etc. This means that all leading spaces MUST be preserved as part of the value and all trailing spaces are considered to be optional whitespace characters not part of the value. Optional whitespace MAY be excluded when propagating the header.
```

This `[[!RFC0020]]` link does indeed look odd on Github and might look like a typo but it's actually used on purpose throughout the document. In the document published by W3C, this adds the referenced RFC to the bibliography on the bottom and adds a link to that: https://www.w3.org/TR/trace-context/#value

Reply:
ah good catch I missed the link being broken